docs/source/tutorial_external_solver.rst

```diff
@@ -0,0 +1,383 @@
+          assert isinstance(op, DiffusionOperator)
+          self.op = op
+          self.source = WrappedVectorSpace(op.dim_source)
+          self.range = WrappedVectorSpace(op.dim_range)
+          self.linear = True
+
+      @classmethod
+      def create(cls, n, left, right):
+          return cls(DiffusionOperator(n, left, right))
+
+      def apply(self, U, mu=None):
+          assert U in self.source
+
+          def apply_one_vector(u):
+              v = Vector(self.range.dim, 0)
+              self.op.apply(u._impl, v)
+              return v
+
+          return self.range.make_array([apply_one_vector(u) for u in U._list])
+
+
+Putting it all together
+-----------------------
+
+As a demonstration we will use our (stationary) toy diffusion solver in a transient example with timestepping
+provided by pyMOR. For details on the discretization setup see :doc:`tutorial01`. First up we implement
+a `discretize` function that uses the `WrappedDiffusionOperator` and `WrappedVectorSpace` to assemble an
+|InstationaryModel|.
+
+.. jupyter-execute::
+
+  from pymor.algorithms.pod import pod
+  from pymor.algorithms.timestepping import ExplicitEulerTimeStepper
+  from pymor.discretizers.builtin.gui.visualizers import OnedVisualizer
+  from pymor.models.basic import InstationaryModel
+  from pymor.discretizers.builtin import OnedGrid
+  from pymor.operators.constructions import VectorOperator, LincombOperator
+  from pymor.parameters.functionals import ProjectionParameterFunctional
+  from pymor.reductors.basic import InstationaryRBReductor
+
+
+  def discretize(n, nt, blocks):
+      h = 1. / blocks
+      ops = [WrappedDiffusionOperator.create(n, h * i, h * (i + 1)) for i in range(blocks)]
+      pfs = [ProjectionParameterFunctional('diffusion_coefficients', blocks, i) for i in range(blocks)]
+      operator = LincombOperator(ops, pfs)
+
+      initial_data = operator.source.zeros()
+
+      # use to_numpy method of WrappedVector to setup rhs
+      # note that we cannot use the data property of ListVectorArray,
+      # since ListVectorArray will always return a copy
+      rhs_vec = operator.range.zeros()
+      rhs_data = rhs_vec._list[0].to_numpy()
+      rhs_data[:] = np.ones(len(rhs_data))
+      rhs_data[0] = 0
+      rhs_data[len(rhs_data) - 1] = 0
+      rhs = VectorOperator(rhs_vec)
+
+      # we can re-use pyMOR's builtin grid and visualizer for our demonstration
+      grid = OnedGrid(domain=(0, 1), num_intervals=n)
+      visualizer = OnedVisualizer(grid)
+
+      time_stepper = ExplicitEulerTimeStepper(nt)
+
+      fom = InstationaryModel(T=1e-0, operator=operator, rhs=rhs, initial_data=initial_data,
+                              time_stepper=time_stepper, num_values=20,
+                              visualizer=visualizer, name='C++-Model')
+      return fom
+
+Now we can build a reduced basis for our model. Note that this code is not specific to our wrapped classes. Those
+only directly used in the `discretize` call.
```

Comment:
Something is missing in the last sentence.

Reply:
'Those wrapped types are only directly used' maybe?

Reply:
Sure.